x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/snapshot/upgrader/SnapshotUpgradePredicate.java

```diff
@@ -11,6 +11,7 @@
 import org.elasticsearch.ElasticsearchException;
 import org.elasticsearch.persistent.PersistentTasksCustomMetadata;
 import org.elasticsearch.xpack.core.ml.job.snapshot.upgrade.SnapshotUpgradeState;
+import org.elasticsearch.xpack.core.ml.job.snapshot.upgrade.SnapshotUpgradeTaskParams;
```

Comment:
```suggestion
```
Unnecessary?

Reply:
I moved `SnapshotUpgradeTaskParams` into the `core` plugin so that it's accessible to `MlTasksTests`.  It used to be in the same package as `SnapshotUpgradePredicate` so could be used without importing but now it's not.